docs/spfx/build-a-hello-world-web-part.md

```diff
@@ -1,171 +1,160 @@
+   
+   Congratulations! You have just added your first client-side web part to a client-side page.
+   
+3. Now, choose the pencil icon on the far left of the web part to reveal the web part property pane.
+   
+   ![HelloWorld web part property pane](./images/sp-workbench-helloworld-pp.png)
+   
+   The property pane is where you can define properties to customize your web part. The property pane is client-side driven and provides a consistent design across SharePoint. 
+   
+4. Modify the text in the **Description** text box to **Client-side web parts are awesome!**
 
-Notice how the text in the web part also changes as you type. 
+   Notice how the text in the web part also changes as you type. 
 
-One of the new capabilities available to the property pane is to configure its update behavior, which can be set to either reactive or non-reactive. By default, the update behavior is reactive and enables you to see the changes as you edit the properties. The changes are saved instantly as well when the behavior is configure to be reactive.  
+One of the new capabilities available to the property pane is to configure its update behavior, which can be set to reactive or non-reactive. By default the update behavior is reactive and enables you to see the changes as you edit the properties. The changes are saved instantly as when the behavior is reactive.  
 
 ## Step 5: Web part project structure
-Switch back to Visual Studio Code to explore the web part project structure. You can find the web part code under the `src\webparts\helloWorld` directory. 
+You can use Visual Studio Code to explore the web part project structure. 
+1. In the console, go to the **src\webparts\helloWorld** directory. 
+2. Enter the following command to open the web part project in Visual Studio Code (or use your favorite editor):
+
+   ```
+   code .
+   ```
 
-![HelloWorld project structure](./images/helloworld-wp-vscode-project-structure.png)
+   ![HelloWorld project structure](./images/helloworld-wp-vscode-project-structure.png)
 
-TypeScript is the primary language to build SharePoint client-side web parts. TypeScript is a typed superset of JavaScript that compiles to plain JavaScript. SharePoint client-side development tools are built using TypeScript classes, modules and interfaces to help developers build robust client-side web parts. 
+   If you get an error, you may need to [install the code command in PATH](https://code.visualstudio.com/docs/editor/setup).
 
-Let's walk through some of the key files in the project:
+TypeScript is the primary language to build SharePoint client-side web parts. TypeScript is a typed superset of JavaScript that compiles to plain JavaScript. SharePoint client-side development tools are built using TypeScript classes, modules, and interfaces to help developers build robust client-side web parts. 
 
-### HelloWorldWebPart.ts
+Let's walk through some of the key files in the project.
```

Comment:
Avoid first person plural. You could revise as "The following are some of the key files in the project."